releasenotes/notes/callback-for-NN-classifier-regressor-8e1d5c887335e6d9.yaml

```diff
@@ -0,0 +1,11 @@
+---
+features:
+  - |
+    The classes ``NeuralNetworkClassifier`` and ``NeuralNetworkRegressor`` has a new optional argument ``callback`` 
```

Comment:
```suggestion
    The classes ``NeuralNetworkClassifier`` and ``NeuralNetworkRegressor`` have a new optional argument ``callback`` 
```

Reply:
Had to make changes in the release note anyway, so rectified in https://github.com/Qiskit/qiskit-machine-learning/pull/151/commits/2a8dbf61e6052a005a73e100da658ed1b976a5fe